File: Stack.cpp
//Stack implementation using cpp
#include <iostream>
using namespace std;
int main() {
    //max size of stack defined
    int maxs=5;
    int s[maxs];
    int top=-1;
    int i=1;int in;int k;
    while (i>0){
        cout<<"\nEnter choice :\n1 to push\n2 to pop\n3 to peek\n0 to exit\n";
        cin>>i;
        switch(i){
            case 1:
                 if (top==(maxs-1)){
                     cout<<"\nStack is full";
                                }
                 else{
                     top++;
                     cout<<"\nEnter Element to be pushed : ";
                     cin>>in;
                     s[top]=in;
                     cout<<"\n"<<in<<" pushed at position "<<(top);
                      }
                 break;
             case 2:
                 if (top==-1){
                     cout<<"\nStack Empty";
                            }
                else{
                    cout<<"\n"<<s[top]<<" POP from position "<<top;
                    top--;
                    }
                    break;
            case 3:
                if (top!=-1){
                cout<<"\nTop element = "<<s[top];}
                else{
                    cout<<"\nStack Enmpty";
                }
                break;
             }
        }
       
}

File: Stack.py
global stack
stack=[]
class s:
    def push(a):
        stack.append(a)
    def pop():
        stack.remove[len(stack)-1]
    def peek():
        return stack[len(stack)-1]
    def size():
        return len(stack)
i=5
while i>0:
    i=int(input("Enter choice\n1 to push\n2 to pop\n3 to peek\n4 to check size\n0 to exit\n"))
    if i==1:
        k=input("Enter element to add : ")
        s.push(k)
        print("PUSH operation successfull")
    elif i==2:
        s.remove()
        print("POP operation successfull")
    elif i==3:
        print(f"Top element ={s.peek()}")
    elif i==4:
        print(f"size of stack={s.size()}")
    elif i!=0:
        print("Invalid Choice")
    
